src/Tizen.Multimedia.Remoting/WebRTC/WebRTCDataChannel.Events.cs

```diff
@@ -132,6 +134,31 @@ public partial class WebRTCDataChannel
             }
         }
 
+        /// <summary>
+        /// Occurs when the buffered amount is lower than <see cref="BufferedAmountLowThreshold"/>.<br/>
+        /// If <see cref="BufferedAmountLowThreshold"/> is not set, this event will not occurred.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        public event EventHandler<EventArgs> BufferedAmountLow
```

Comment:
I'm not convinced about the event name, nothing in that name suggests it's an event. Unfortunately, the names I can think of like `BufferedAmountDroppedBelowThreshold` or `BufferedAmountFallenBelowTheshold` or `BufferDrained` are really better, so I think I will accept the current name as it is.

Reply:
The word `BufferedAmountLow` is used in the WebRTC spec. So we want to follow it.

Reply:
ok, let's keep it as it is